vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
@StephenHynes7 don't know, I need some suggestions on how to fix this.

BTW: I don't think it is a common thing to change the certificate, and, I presume, other log drivers running over HTTPS will also need the certificate somehow, am I right?